pkg/handlers/embedded_cluster_node_join_command.go

```diff
@@ -163,6 +166,17 @@ func (h *Handler) GetEmbeddedClusterNodeJoinCommand(w http.ResponseWriter, r *ht
 		ecVersion = install.Spec.Config.Version
 	}
 
+	airgapRegistryAddress := ""
+	if install.Spec.AirGap {
+		airgapRegistrySvc, err := client.CoreV1().Services("registry").Get(r.Context(), "registry", metav1.GetOptions{})
```

Comment:
You can use the `GetEmbeddedRegistryCreds` that we use elsewhere which would return the port number too I think. Example: https://github.com/replicatedhq/kots/blob/ea2056ef211a8940481dd500b5a7913a2ad1cd24/pkg/kotsadmsnapshot/backup.go#L387